src/directives/index.esm.js

```diff
@@ -0,0 +1,18 @@
+// Index file used for the main builds, which does not include legacy plugin names
+import { installFactory } from '../utils/plugins'
+import * as directivePlugins from './plugins'
+
+// Export all directive group plugins as named exports (VB<name>Plugin)
+export * from './plugins'
+
+// Named exports of all directives (VB<Name>)
+export * from './toggle'
+export * from './modal'
+export * from './scrollspy'
+export * from './tooltip'
+export * from './popover'
+
+// Default export is a plugin that installs all plugins
+export default {
```

Comment:
As this is a new export, I think would be better to always prefer named exports. (And using `plugin` named export)

Reply:
OK cool... that make sense.

Reply:
Actually maybe not...

as we are re-exporting all named exports... and need to avoid duplicated... so maybe directivesPlugin should be the name